saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -53,8 +53,19 @@ class Meta:
         error_type_field = "product_errors"
 
     @classmethod
-    def bulk_action(cls, queryset):
-        delete_categories(queryset.values_list("pk", flat=True))
+    def perform_mutation(cls, _root, info, ids, **data):
```

Comment:
Why do we override `perform_mutation` here? To pass the manager as an extra argument? It is available in the `info` object anyway.

Reply:
I override this method because, I wanted to pass manager to `bulk_action` method.
`perform_mutation` method in BaseBulkMutation takes `**data` parameter and pass it to `bulk_action`. So I added managet to `data` argument.

Reply:
So maybe we should extend bulk_action to handle info by default? All future bulk mutations which use manager will require the same action,

Reply:
Yep, was going to write the same as what Maciek suggests.

Reply:
Ok, I will change that.

Reply:
I already added info to core `bulk_action` in #7052, it looks like we can merge it, so the problem will be solved.